server/websocket/src/connection.rs

```diff
@@ -69,13 +71,106 @@ impl From<OutgoingMessage> for WsMessage {
     }
 }
 
+/// Helper function to send outgoing messages.
+fn send_message(
+    sink: &mpsc::UnboundedSender<WsMessage>,
+    msg: OutgoingMessage,
+) -> Result<(), WsError> {
+    sink.unbounded_send(msg.into())
+        .map_err(|_| WsError::Http(500))
+}
+
+/// Helper function to send error messages.
+fn send_error_string(
+    sink: &mpsc::UnboundedSender<WsMessage>,
+    id: String,
```

Comment:
Let's be more obvious here and call the two parameters `operation_id` and `err_str` or something.